frontend/packages/kubevirt-plugin/integration-tests-cypress/types/vm.ts

```diff
@@ -13,6 +13,9 @@ export type Disk = {
   };
   source?: DISK_SOURCE;
   bootable?: boolean;
+  autoDetach?: boolean;
+  url?: string;
```

Comment:
'url' and 'pvc' may no need by using `enums/provisionSource.ts`.

Reply:
Right. I'll fix it